### PR TITLE
Added refreshing the page after changing the domain.

### PR DESCRIPTION
**Pull request by BasilSnowman**
Added refreshing the page after changing the domain (for real info on the configuration).

### CODE REVIEW

File: domain_config_ui/src/Form/SwitchForm.php [hunk 1]
```diff
@@ -79,11 +110,20 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
...
    * @param array $form
+   *   The form array.
    * @param FormStateInterface $form_state
+   *   The form state array.
+   * @return AjaxResponse
+   *   The response.
    */
   public function switchCallback(array &$form, FormStateInterface $form_state) {
     $this->domainNegotiator->setSelectedDomain($form_state->getValue('config_save_domain'));
+    $url = Url::createFromRequest($this->requestStack->getCurrentRequest());
```
Comment (andypost) [inline domain_config_ui/src/Form/SwitchForm.php]: Are you sure that current request is the same as current page has?

**Review by andypost — CHANGES_REQUESTED** [review]
Current request is the main question

File: domain_config_ui/src/Form/SwitchForm.php [hunk 2]
```diff
@@ -79,11 +110,20 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
    * Callback to remember save mode.
    *
    * @param array $form
+   *   The form array.
    * @param FormStateInterface $form_state
+   *   The form state array.
+   * @return AjaxResponse
```
Comment (andypost) [inline domain_config_ui/src/Form/SwitchForm.php]: Needs new line before for code style

Comment (BasilSnowman) [reply domain_config_ui/src/Form/SwitchForm.php]: fixed

File: domain_config_ui/src/Form/SwitchForm.php [hunk 3]
```diff
@@ -79,11 +108,24 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
...
+   *   The response.
    */
   public function switchCallback(array &$form, FormStateInterface $form_state) {
     $this->domainNegotiator->setSelectedDomain($form_state->getValue('config_save_domain'));
     $response = new AjaxResponse();
+    // Refresh the page after changing the domain.
+    $path = $form['current_path']['#value'];
+    if ($path != '') {
+      $url = Url::fromUri('base:' . $path);
```
Comment (andypost) [inline domain_config_ui/src/Form/SwitchForm.php:124]: Needs to call setAbsolute() before redirect, isn't it?

Comment (BasilSnowman) [reply domain_config_ui/src/Form/SwitchForm.php:124]: yes, you right

**Pull request merged**